sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/SchemaRegistryClientLiveTests.cs

```diff
@@ -23,22 +23,50 @@ public SchemaRegistryClientLiveTests(bool isAsync, SchemaRegistryClientOptions.S
             _serviceVersion = version;
         }
 
-        private SchemaRegistryClient CreateClient(string format)
+        private SchemaRegistryClient CreateClient(string format, int group = 1)
         {
             string endpoint;
             switch (format)
             {
                 case Json:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointJson;
+                    if (_serviceVersion == SchemaRegistryClientOptions.ServiceVersion.V2022_10)
+                    {
+                        endpoint = (group == 1) ? TestEnvironment.SchemaRegistryEndpointJson : TestEnvironment.SchemaRegistryEndpointJson2;
+                    }
+                    else
+                    {
+                        endpoint = (group == 1) ? TestEnvironment.SchemaRegistryEndpointJson2021 : TestEnvironment.SchemaRegistryEndpointJson2021_2;
+                    }
                     break;
                 case Avro:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointAvro;
+                    if (_serviceVersion == SchemaRegistryClientOptions.ServiceVersion.V2022_10)
+                    {
+                        endpoint = (group == 1) ? TestEnvironment.SchemaRegistryEndpointAvro : TestEnvironment.SchemaRegistryEndpointAvro2;
+                    }
+                    else
+                    {
+                        endpoint = (group == 1) ? TestEnvironment.SchemaRegistryEndpointAvro2021 : TestEnvironment.SchemaRegistryEndpointAvro2021_2;
+                    }
                     break;
                 case Custom:
-                    endpoint = TestEnvironment.SchemaRegistryEndpointCustom;
+                    if (_serviceVersion == SchemaRegistryClientOptions.ServiceVersion.V2022_10)
+                    {
+                        endpoint = (group == 1) ? TestEnvironment.SchemaRegistryEndpointCustom : TestEnvironment.SchemaRegistryEndpointCustom2;
```

Comment:
Do we actually need multiple endpoints per service version? I thought it was just that we would need 1 for each service version. But as mentioned in this [comment](https://github.com/Azure/azure-sdk-for-net/pull/33820/files?short_path=0b58de4#r1095243687), I think we can just use a single namespace and create 6 schema groups within it.